wheel_left_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel_left

wheel_right_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel_right